src/xscen/extract.py

```diff
@@ -1068,6 +1035,174 @@ def _get_warming_level(model):
     return out
 
 
+@parse_config
+def get_warming_level_from_period(
+    realization: (
+        xr.Dataset | xr.DataArray | dict | pd.Series | pd.DataFrame | str | list
+    ),
+    period: list[str],
+    *,
+    tas_baseline_period: Sequence[str] | None = None,
```

Comment:
since the first argument is `period`, perhaps the second argument should simply be `baseline_period`?

Reply:
My idea was to keep the arguments pretty much the same between `get_warming_levels` and this new function.